libnavigation-ui/src/main/java/com/mapbox/navigation/ui/map/NavigationMapboxMap.java

```diff
@@ -430,41 +453,49 @@ public void removeProgressChangeListener() {
    * This method uses {@link NavigationMapboxMapInstanceState}, stored with the provided key.  This key
    * can also later be used to extract the {@link NavigationMapboxMapInstanceState}.
```

Comment:
Should we remove this as `NavigationMapboxMapInstanceState` is not publicly available anymore?